plugins/reporters/evaluated-model/src/main/kotlin/EvaluatedModelMapper.kt

```diff
@@ -426,21 +426,19 @@ internal class EvaluatedModelMapper(private val input: ReporterInput) {
                 calculatePackageVerificationCode(fileList.files.map { it.sha1 }.asSequence())
             }.orEmpty(),
             issues = issues
-        )
-
-        val actualScanResult = scanResults.addIfRequired(evaluatedScanResult)
+        ).run { scanResults.addIfRequired(this) }
```

Comment:
Hmm... but shouldn't `evaluatedScanResult` in line 420 be renamed to `actualScanResult` then? Because `addIfRequired()` might not add the `EvaluatedScanResult` at all, so it's a bit odd to still call it `evaluatedScanResult` in any case, or?

Doing the rename would also reduce the diff below.

Reply:
I do not really see the problem with calling it `evaluatedScanResult`.

Reply:
Ah, now I see what confused me: Your first commit renames the `result` function parameter (of type `ScanResult`) to `scanResult`, but there is also the `scanResults` member variable (plural) which is a list of `EvaluatedScanResult`. As the only difference between those variables is the singular vs. plural naming, I thought they would be of the same type (`ScanResult`), which is why I found the name misleading.

From that perspective this particular renaming is ok for me now. Though I still feel that `scanResult` and `scanResults` not being of the same base type is a bit misleading.

Reply:
I've dropped the commit which renamed `result`.